### PR TITLE
Add array validation docs

### PR DESCRIPTION
**Pull request by romanov-acc**
Add docs for https://github.com/octobercms/library/pull/316

### CODE REVIEW

File: database-attachments.md [hunk 1]
```diff
@@ -166,3 +167,32 @@ Example:
     $user = $file->attachment;
     
 For more information read the [polymorphic relationships](../database/relations#polymorphic-relations)
+
+<a name="attachments-validation-example"></a>
+### Validation example
+
+This section shows how you can validate `attachMany` attachments.
+
+~~~php
```
Comment (LukeTowers) [inline database-attachments.md]: Code examples aren't enclosed with `~~~php`, they just need to be indented by four spaces

File: database-attachments.md [hunk 2]
```diff
@@ -166,3 +167,32 @@ Example:
...
+    public $rules = [
+        'photos'   => 'required',
+        'photos.*' => 'image|max:1000|dimensions:min_width=100,min_height=100'
+    ];
+
+    /* some other code */
+}
+~~~
+
+For more information read the [validating arrays](../services/validation#validating-arrays)
```
Comment (LukeTowers) [inline database-attachments.md]: Change this sentence to 

> For more information on the `attribute.*` syntax used above, see [validating arrays](../services/validation#validating-arrays).

File: database-attachments.md [hunk 3]
```diff
@@ -166,3 +167,32 @@ Example:
     $user = $file->attachment;
     
 For more information read the [polymorphic relationships](../database/relations#polymorphic-relations)
+
+<a name="attachments-validation-example"></a>
+### Validation example
+
+This section shows how you can validate `attachMany` attachments.
```
Comment (LukeTowers) [inline database-attachments.md]: Change to 
> The example below uses [array validation](../services/validation#validating-arrays) to validate `$attachMany` relationships.

**Comment by romanov-acc** [comment]
Hi @LukeTowers . Sorry for mistakes. My English not so good :)

**Comment by LukeTowers** [comment]
No worries, thanks for putting this together!

**Pull request merged**